modulegen/dependabot.go

```diff
@@ -39,6 +46,11 @@ func NewUpdate(example Example) Update {
 			Interval: updateSchedule,
 			Day:      "sunday",
 		},
+		Groups: Groups{
+			"all": Group{
+				Patterns: []string{"*"},
```

Comment:
Could you double check that this will generate:
```yaml
      groups:
        all:
          patterns:
            - "*"
```

and no:
```yaml
      groups:
        all:
          patterns:
            - *
```
🙏 

From modules dir:
```shell
go run . --name questdb --image "questdb/questdb:latest" --title QuestDB --as-module
```

Reply:
Ok, that shall be alright